src/js/stores/SourceStore.js

```diff
@@ -0,0 +1,53 @@
+// src/stores/ContactStore.js
```

Comment:
no need to annotate the file with its name. Naming the file is descriptive enough.